/signal_generator:
  ros__parameters:
    amplitud: 0.5
    offset: 0.5
    periodo: 1.0
    phase: 0.0
    sampling_time: 0.05
    signal_type: 4
    use_sim_time: false

